reviewing/plugin-commands-licenses/src/outputRenderer.ts

```diff
@@ -112,13 +112,46 @@ function renderLicensesTable (
   for (let i = 0; i < columnNames.length; i++)
     columnNames[i] = chalk.blueBright(columnNames[i])
 
+  const data = [
+    columnNames,
+    ...deduplicateLicensesPackages(sortLicensesPackages(licensePackages))
+      .map((licensePkg) => columnFns.map((fn) => fn(licensePkg))),
+  ]
+  let detailsColumnMaxWidth = 40
+  let packageColumnMaxWidth = 0
+  let licenseColumnMaxWidth = 0
+  if (opts.long) {
+    detailsColumnMaxWidth = data.slice(1).reduce((maxWidth, row) => {
+      const texts = row[2].split('\n')
+      const link = texts.find((line) => line.startsWith('https'))
+      // Use the package link to determine the width of the details column
+      const detailMaxWidth = link ? link.length : 0
+      const repeatCount = Math.max(0, texts.length - 1)
+      row[0] = `${row[0]}${'\n '.repeat(repeatCount)}` // Add extra lines to the package column
+      row[1] = `${row[1]}${'\n '.repeat(repeatCount)}` // Add extra lines to the license column
+      packageColumnMaxWidth = Math.max(packageColumnMaxWidth, row[0].length)
+      licenseColumnMaxWidth = Math.max(licenseColumnMaxWidth, row[1].length)
+      return Math.max(maxWidth, detailMaxWidth)
+    }, 0)
+    const remainColumnWidth = process.stdout.columns - packageColumnMaxWidth - licenseColumnMaxWidth - 20
```

Comment:
At first I though that just subtracting 4 would be enough, but I found that the following error would occur after doing so, so I decided to leave more space.

<img width="609" alt="image" src="https://github.com/pnpm/pnpm/assets/24516654/34e35971-2b08-45ea-8591-c8b257c8caf8">